app/assets/javascripts/angular/controllers/votes_controller.coffee

```diff
@@ -10,7 +10,11 @@ SupportController = [ '$scope', '$location', '$rootScope', 'Vote', ( $scope, $lo
       ($scope.vote
       ), (( response, status, headers, config ) ->
         $rootScope.$broadcast 'event:votesChanged'
-        $scope.alertService.setSuccess 'Your vote was created with the comment: \"' + response.comment + '\"', $scope, 'main'
+        if response.comment is null
+          responseMessage = "Your vote was recorded without a comment"
```

Comment:
While your assignment and usage of ```responseMessage``` does work, I think it's technically updated out of scope from your final usage. In any case, my editor complains. The fix would be to insert at line 12: ```responseMessage = undefined```